assets/js/custom/TranslateProgram.js

```diff
@@ -0,0 +1,135 @@
+    $(document).on('click', '#program-translation-button', function () {
+      $(this).hide()
+
+      if (translateProgram.isTranslationNotAvailable('#name-translation')) {
+        $('#program-translation-loading-spinner').show()
+        translateProgram.translateProgram()
+      } else {
+        translateProgram.openTranslatedProgram()
+      }
+    })
+
+    $(document).on('click', '#remove-program-translation-button', function () {
+      $(this).hide()
+      $('#program-translation-button').show()
+
+      $('#name').removeClass('program-name').addClass('program-name-animation')
+      $('#name-translation').removeClass('program-name').addClass('program-name-animation')
+      $('#name-translation').animate({ width: 'toggle' })
+      $('#name').animate({ width: 'toggle' }, translateProgram.ANIMATION_TIME,
+        function () {
+          $('#name').removeClass('program-name-animation').addClass('program-name')
+          $('#name-translation').removeClass('program-name-animation').addClass('program-name')
+        }
+      )
+
+      if (translateProgram.hasDescription) {
+        $('#description').slideDown()
+        $('#description-translation').slideUp()
+      }
+
+      $('#credits-translation-wrapper').slideUp()
+      if (translateProgram.hasCredit) {
+        $('#credits').slideDown()
+      }
+    })
+  }
+
+  setTranslatedProgramData (data) {
+    $('#name-translation').attr('lang', data.target_language)
+    $('#name-translation').text(data.translated_title)
+
+    if (this.hasDescription) {
+      $('#description-translation').text(data.translated_description)
+    }
+
+    if (this.hasCredit) {
+      $('#credits-translation').text(data.translated_credit)
+    }
+
+    const byLineElements = new ByLineElementContainer(
+      '#program-translation-before-languages',
+      '#program-translation-between-languages',
+      '#program-translation-after-languages',
+      '#program-translation-first-language',
+      '#program-translation-second-language'
+    )
+
+    this.setTranslationCredit(data, byLineElements)
+  }
+
+  openTranslatedProgram () {
+    $('#program-translation-loading-spinner').hide()
+    $('#remove-program-translation-button').show()
+
+    $('#name').removeClass('program-name').addClass('program-name-animation')
+    $('#name-translation').removeClass('program-name').addClass('program-name-animation')
+    $('#name').animate({ width: 'toggle' })
+    $('#name-translation').animate({ width: 'toggle' }, this.ANIMATION_TIME,
+      function () {
+        $('#name').removeClass('program-name-animation').addClass('program-name')
+        $('#name-translation').removeClass('program-name-animation').addClass('program-name')
+      }
+    )
+
+    if (this.hasDescription) {
+      $('#description-translation').slideDown()
+      $('#description').slideUp()
+    }
+
+    $('#credits-translation-wrapper').slideDown()
+    if (this.hasCredit) {
+      $('#credits').slideUp()
+    }
```

Comment:
So that this can be changed to 

```
if (this.hasCredit) {
      $('#credits-translation').slideDown()
      $('#credits').slideUp()
}
```

Mirroring the description part.

Then add `$('#credit-byline').slideDown()` separately.